test/Combinatorics/Graph.jl

```diff
@@ -4,6 +4,7 @@
         g = Graph{Directed}(5)
         @test n_vertices(g) == 5
         @test n_edges(g) == 0
+        @test vertices(g) == 1:5
```

Comment:
```suggestion
        @test issetequal(vertices(g), 1:5)
```
I think it would be better to compare these as sets, as the docstring does not guarantee the return type to be `UnitRange{Int}`.

Reply:
I wasn't aware that there is a dispatch for that. Yes, this is fine then.